panic: interface conversion with ldap server

### Interactsh version:
main | dev

### Current Behavior:

```console
1.7054656536116006e+09  info    maintenance     advancing OCSP staple   {"identifiers": ["hackwithautomation.com"], "from": 1705768018, "to": 1705974658}
panic: interface conversion: message.ProtocolOp is message.DelRequest, not message.CompareRequest

goroutine 1478620 [running]:
github.com/Mzack9999/ldapserver.(*Message).GetCompareRequest(...)
        /root/go/pkg/mod/github.com/!mzack9999/ldapserver@v1.0.2-0.20211229000134-b44a0d6ad0dd/message.go:50
github.com/projectdiscovery/interactsh/pkg/server.(*LDAPServer).handleDelete(0x4121e5?, {0x1386880?, 0xc000049280?}, 0x10?)
        /root/interactsh/pkg/server/ldap_server.go:269 +0x3fb
github.com/Mzack9999/ldapserver.(*RouteMux).ServeLDAP(0xc0007c2ee0, {0x1386880, 0xc000049280}, 0xc00029cb10)
        /root/go/pkg/mod/github.com/!mzack9999/ldapserver@v1.0.2-0.20211229000134-b44a0d6ad0dd/route.go:150 +0xb8
github.com/Mzack9999/ldapserver.(*client).ProcessRequestMessage(0xc00007e680, 0xc00036ef00)
        /root/go/pkg/mod/github.com/!mzack9999/ldapserver@v1.0.2-0.20211229000134-b44a0d6ad0dd/client.go:240 +0x18a
created by github.com/Mzack9999/ldapserver.(*client).serve in goroutine 1478629
        /root/go/pkg/mod/github.com/!mzack9999/ldapserver@v1.0.2-0.20211229000134-b44a0d6ad0dd/client.go:160 +0x6d3
```

--- a/pkg/server/ldap_server.go
+++ b/pkg/server/ldap_server.go
@@ -266,10 +266,10 @@ func (ldapServer *LDAPServer) handleAdd(w ldap.ResponseWriter, m *ldap.Message) 
 func (ldapServer *LDAPServer) handleDelete(w ldap.ResponseWriter, m *ldap.Message) {
 	atomic.AddUint64(&ldapServer.options.Stats.Ldap, 1)
 
-	r := m.GetCompareRequest()
+	r := m.GetDeleteRequest()
 	var message strings.Builder
 	message.WriteString("Type=Delete\n")
-	message.WriteString(fmt.Sprintf("Entity=%s\n", r.Entry()))
+	message.WriteString(fmt.Sprintf("Entity=%s\n", r))
 
 	res := ldap.NewDeleteResponse(ldap.LDAPResultSuccess)
 	w.Write(res)

